### DISCUSSION TITLE
ImportError for lemmy.pipe

### DISCUSSION THREAD

**Discussion by vrunm** [explosion/spaCy#12224] [Help: Coding & Implementations] [answered]
I'm trying to use Lemmy as a SpaCy integration following the instructions [here.](https://spacy.io/universe/project/lemmy) However, when I try running import lemmy.pipe I get the error ImportError: cannot import name 'PRON_LEMMA' from 'spacy.symbols' .

I'm at a loss - how do I get around this?

I'm using python 3.8 and SpaCy 3.3.0 with the [DaCy pipeline](https://pythonawesome.com/dacy-the-state-of-the-art-danish-nlp-pipeline-using-spacy/).

Thank you for you input.

**Comment by rmitsch** [comment]
Hi @vrunm, a note: we don't maintain third-party packages, so we have limited insight into this. That said, can you provide a minimal reproducible example and the versions of `Lemmy` and `DaCy` you're using?

**Comment by vrunm** [comment]
I found an answer to this over here: https://github.com/explosion/spaCy/discussions/11009 
It seems this only works on spacy v2 .
After downgrading to spacy 2.1.0 it still gives the same import error.

**Comment by vrunm** [comment]
I am using Spacy v2.0 and AllenNlp v1.5.0 
I am running into this import issue:

`cannot import name 'PRON_LEMMA' from 'spacy.symbols' (/opt/conda/lib/python3.7/site-packages/spacy/symbols.cpython-37m-x86_64-linux-gnu.so)`

**Comment by rmitsch** [comment, marked as answer]
I recommend reaching out to the folks at [DaCy](https://github.com/centre-for-humanities-computing/DaCy). The project seems to be actively maintained and they are more likely able to help you out with this.
